cocos/scene-graph/node.ts

```diff
@@ -1472,8 +1475,12 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
     protected _dirtyFlags = TransformBit.NONE; // does the world transform need to update?
     protected _eulerDirty = false;
 
-    protected _flagChangeVersion = 0;
-    protected _hasChangedFlags = 0;
+    /**
+     * The high bits are used to store the version number of the changedFlag, and the low 3 bits represent its specific value
+     *
+     * | 31 - 29 reserved | 28 - 3 version number | 2  - 0 : Scale Rotation Translation|
+     */
+    protected _hasChangedFlagsWithVersion = 0;
```

Comment:
May be `_changeFlagAndRTS` is a better name.

Reply:
Done